recipe/meta.yaml

```diff
@@ -1,41 +1,44 @@
 {% set name = "langchain" %}
-{% set version = "0.0.142" %}
+{% set version = "0.0.157" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/langchain-{{ version }}.tar.gz
-  sha256: cc86e1ed5656380f4928f17bb0130281d42d68280c003b04ea7eb890dbdbb479
+  sha256: e93411664b09988ad100c807ff3f8eb118e053d926878fd4e3cbaa0f2279a467
 
 build:
+  skip: True # [py<38]
   entry_points:
     - langchain-server = langchain.server:main
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
 
 requirements:
   host:
-    - python >=3.8,<4.0
-    - poetry-core
     - pip
+    - wheel
+    - python
+    - poetry-core >=1.0.0
   run:
-    - gptcache >=0.1.7
-    - numexpr >=2.8.4,<3.0.0
-    - python >=3.8.1,<4.0
+    - python
     - pydantic >=1.0.0,<2.0.0
-    - sqlalchemy >=1.0.0,<2.0.0
+    - sqlalchemy >=1.3,<3
     - requests >=2.0.0,<3.0.0
     - pyyaml >=5.4.1
     - numpy >=1.0.0,<2.0.0
     - openapi-schema-pydantic >=1.2.0,<2.0.0
     - dataclasses-json >=0.5.7,<0.6.0
     - tenacity >=8.1.0,<9.0.0
     - aiohttp >=3.8.3,<4.0.0
-    - async-timeout >=4.0.0,<5.0.0
+    - async-timeout >=4.0.0,<5.0.0  # [py<311]
+    - numexpr >=2.8.4,<3.0.0
+    - tqdm >=4.48.0
```

Comment:
`tqdm >=4.48.0` is optional too https://github.com/hwchase17/langchain/blob/b67be55ab8187c9b1ac09df501fbe698e08711bb/pyproject.toml#L21